entities/Rules/index.js

```diff
@@ -404,8 +409,74 @@ module.exports = {
       notify('MORTGAGE');
     },
   ],
+  RESOLVE_SPECIAL_PROPERTY: [
+    ({ UI, notify }, gameState) => {
+      const boardProperty = gameState.currentBoardProperty;
+      if (boardProperty.id.includes('chance')) {
```

Comment:
I think this is addressed in an open PR, but can we pop a `// TODO:` in here, just in case ? But it is moot

Reply:
Going to leave as is as it will be fixed soon, but good note for the future!